modules/constructor/constructor-basic/constructor-basic.model.js

```diff
@@ -1,17 +1,21 @@
 const mongoose = require('mongoose');
 const Language = require('../../../models/Language').schema;
 const CurrencySet = require('../../../models/CurrencySet').schema;
-const ImageSet = require('../../common/ImageSet').schema;
 
 const constructorBasicSchema = new mongoose.Schema({
   name: [Language],
   material: {
     type: mongoose.Schema.Types.ObjectId,
     ref: 'Material',
   },
+  color: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Color',
+  },
   image: String,
   basePrice: [CurrencySet],
-  available: Boolean
+  available: Boolean,
+  default:Boolean,
```

Comment:
is this `default` as key or you want set some field with default value?

Reply:
Thit id default as a key in order you want use this elemend by defauld when counstructor is loaded for the first time